examples/tcpblaster/tcpblaster_server.c

```diff
@@ -257,7 +257,7 @@ void tcpblaster_server(void)
 
           fkbsent  = (float)recvtotal / 1024.0;
           felapsed = (float)elapsed.tv_sec + (float)elapsed.tv_nsec / 1000000000.0;
-            printf("[%s] %d: Received %d buffers: %7.1f KB (buffer average size: %5.1f KB) in %6.2f seconds (%7.1f KB/second)\n",
+          printf("[%s] %d: Received %d buffers: %7.1f KB (buffer average size: %5.1f KB) in %6.2f seconds (%7.1f KB/second)\n",
```

Comment:
Let's merge into one patch instead

Reply:
@xiaoxiang781216 Do you mean, squash all the commits into one so master doesn't end up with multiple commits?

Reply:
No, it doesn't make sense to request all patches merge into one. But in one PR, if the first patch has some issue, and the second patch fix that issue(.e.g printf alignment, printf("=")), we should merge the second patch into the first one, instead make the mainline history urgly.
Each patch should work well individual without the following patch.

Reply:
@xiaoxiang781216 Ok, I'll look into how to do what you ask!

Reply:
It's very easy, all is automatically with git rebase --interactive:
https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d
Once your finish to refine the patchset, you can run git push with -f flag to update the PR.